src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -72,13 +73,22 @@ object EasyStageDataset {
     for {
       dataDir <- getDataDir
       _ <- mkdirSafe(s.sdoSetDir)
-      emdContent <- createDatasetSdo()
+      (emdContent,amdContent) <- createDatasetSdo()
       _ = log.info("Creating file and folder SDOs")
       _ <- createFileAndFolderSdos(dataDir, DATASET_SDO, emdContent.getEmdRights.getAccessCategory)
-    } yield ()
+    } yield (emdContent,amdContent)
   }
 
   def createFileAndFolderSdos(dir: File, parentSDO: String, rights: AccessCategory)(implicit s: Settings): Try[Unit] = {
+    val maybeSha1Map: Try[Map[String, String]] = Try {
```

Comment:
Nice, but not really necessary, as have presumably already checked that the bag is valid. If one of the manifests were not correctly formatted that would have shown up.